src/main/groovy/nebula/plugin/dependencylock/tasks/MigrateToCoreLocksTask.groovy

```diff
@@ -0,0 +1,129 @@
+/**
+ *
+ *  Copyright 2018 Netflix, Inc.
+ *
+ *     Licensed under the Apache License, Version 2.0 (the "License");
+ *     you may not use this file except in compliance with the License.
+ *     You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *     Unless required by applicable law or agreed to in writing, software
+ *     distributed under the License is distributed on an "AS IS" BASIS,
+ *     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *     See the License for the specific language governing permissions and
+ *     limitations under the License.
+ *
+ */
+
+package nebula.plugin.dependencylock.tasks
+
+import nebula.plugin.dependencylock.ConfigurationsToLockFinder
+import nebula.plugin.dependencylock.DependencyLockReader
+import nebula.plugin.dependencylock.utils.CoreLocking
+import org.gradle.api.BuildCancelledException
+import org.gradle.api.execution.TaskExecutionGraph
+import org.gradle.api.logging.Logger
+import org.gradle.api.logging.Logging
+import org.gradle.api.tasks.InputFile
+import org.gradle.api.tasks.OutputDirectory
+import org.gradle.api.tasks.TaskAction
+
+class MigrateToCoreLocksTask extends AbstractLockTask {
```

Comment:
This is awesome! thanks for adding a task!

Reply:
I'm glad for it, too!